What's going on with the `armv7_simd_test` CI run? Does it have a separate requirements file?

```
_____________ ERROR collecting typing/tests/test_generic_alias.py ______________
ImportError while importing test module '/usr/local/lib/python3.8/dist-packages/numpy-1.24.0.dev0+51.g91819691c-py3.8-linux-armv7l.egg/numpy/typing/tests/test_generic_alias.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/usr/local/lib/python3.8/dist-packages/numpy-1.24.0.dev0+51.g91819691c-py3.8-linux-armv7l.egg/numpy/typing/tests/test_generic_alias.py:13: in <module>
    from typing_extensions import Unpack
E   ModuleNotFoundError: No module named 'typing_extensions'
```

--- a/.github/workflows/build_test.yml
+++ b/.github/workflows/build_test.yml
@@ -276,12 +276,13 @@ jobs:
           git config --global --add safe.directory /numpy
           cd /numpy &&
           python3 setup.py install
+          python3 pip3 install -r ./test_requirements.txt
         "
         docker commit the_build the_build
     - name: Run SIMD Tests
       run: |
         docker run --rm --interactive -v $(pwd):/numpy the_build /bin/bash -c "
-          cd /numpy && python3 pip3 install -r ./test_requirements.txt && python3 runtests.py -n -v -- -k test_simd
+          cd /numpy && python3 runtests.py -n -v -- -k test_simd
         "
 
   sde_simd_avx512_test:

--- a/.github/workflows/build_test.yml
+++ b/.github/workflows/build_test.yml
@@ -276,7 +276,7 @@ jobs:
           git config --global --add safe.directory /numpy
           cd /numpy &&
           python3 setup.py install
-          python3 pip3 install -r ./test_requirements.txt
+          python3 pip install -r ./test_requirements.txt
         "
         docker commit the_build the_build
     - name: Run SIMD Tests

--- a/.github/workflows/build_test.yml
+++ b/.github/workflows/build_test.yml
@@ -281,7 +281,7 @@ jobs:
     - name: Run SIMD Tests
       run: |
         docker run --rm --interactive -v $(pwd):/numpy the_build /bin/bash -c "
-          cd /numpy && python3 runtests.py -n -v -- -k test_simd
+          cd /numpy && python3 pip3 install -r ./test_requirements.txt && python3 runtests.py -n -v -- -k test_simd
         "
 
   sde_simd_avx512_test:

